mp/src/game/client/cdll_client_int.cpp

```diff
@@ -989,6 +989,37 @@ int CHLClient::Init( CreateInterfaceFn appSystemFactory, CreateInterfaceFn physi
 	// Initialize the console variables.
 	ConVar_Register( FCVAR_CLIENTDLL );
 
+    //Remove cheat flag for this cvar
+    ConVar *mat_dynamic_tonemapping = NULL;
+    mat_dynamic_tonemapping = g_pCVar->FindVar("mat_dynamic_tonemapping");
```

Comment:
Declare and initialize on same line:
`ConVar *mat_dynamic_tonemapping = g_pCVar->FindVar("mat_dynamic_tonemapping");`